modules/woocommerce-analytics/classes/wp-woocommerce-analytics-universal.php

```diff
@@ -227,6 +227,7 @@ public function checkout_process() {
 					'pi': '" . esc_js( $item_details['id'] ) . "',
 					'pn': '" . esc_js( $item_details['name'] ) . "',
 					'pc': '" . esc_js( $item_details['category'] ) . "',
+					'pq': '" . esc_js( $item_details['quantity'] ) . "',
```

Comment:
I hadn't included `quantity` is I'm not sure we need it for the conversion funnels; however, we have it and it may be useful in the future. Good work 👍